### PR TITLE
K8s-docs update footer

### PR DESCRIPTION
**Pull request by evilnick**
## Done

- Updates the footer on each page to include a link to the mattermost channel and link to the new contributing page

**NOTE!**
There are several other PRs for K8s docs which will need to be merged first as they introduce new pages
 - https://github.com/canonical/ubuntu.com/pull/13223
 - https://github.com/canonical/ubuntu.com/pull/13267
 - https://github.com/canonical/ubuntu.com/pull/13271

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
    - Be sure to test on mobile, tablet and desktop screen sizes

### CODE REVIEW

File: templates/kubernetes/docs/explain-cloud.md [hunk 1]
```diff
@@ -37,8 +37,9 @@ For more details, see the various cloud integration pages.
 <div class="p-notification--information">
   <div class="p-notification__content">
     <p class="p-notification__message">We appreciate your feedback on the documentation. You can
-    <a href="https://github.com/charmed-kubernetes/kubernetes-docs/edit/main/pages/k8s/explain-cloud.md" >edit this page</a>
+    <a href="https://github.com/charmed-kubernetes/kubernetes-docs/edit/main/pages/k8s/how-to-clouds.md" >edit this page</a>
```
Comment (anthonydillon) [inline templates/kubernetes/docs/explain-cloud.md:40]: It looks like this was changed by mistake

**Review by anthonydillon — CHANGES_REQUESTED** [review]
One small comment in the code.

File: templates/kubernetes/docs/explain-cloud.md (hunk 1 above)
Comment (evilnick) [reply templates/kubernetes/docs/explain-cloud.md]: yup, well spotted :)

Comment (evilnick) [reply templates/kubernetes/docs/explain-cloud.md]: fixed, I think. thanks for checking it over

**Comment by evilnick** [comment]
can we re-run the tests, looks like it can't be merged

**Comment by anthonydillon** [comment]
I think this needs to be rebased on main.

**Comment by evilnick** [comment]
Okay, thanks, will try and do that today and update

**Pull request merged**